database/schema/01-auth.sql

```diff
@@ -1,4 +1,14 @@
 CREATE TABLE account
 (
     address TEXT NOT NULL PRIMARY KEY
+);
+
+
+CREATE TABLE vesting_account
+(
+    address             TEXT        NOT NULL REFERENCES account (address) PRIMARY KEY,
+    original_vesting    JSONB       NOT NULL DEFAULT '{}',
+    end_time            TIMESTAMP   NOT NULL,
```

Comment:
```start_time ``` and```end_time``` should be of type TEXT here, they are both strings: 

<img width="266" alt="Screenshot 2021-10-15 at 12 52 19 PM" src="https://user-images.githubusercontent.com/37407870/137482612-ae4c062a-5ec5-442f-a703-a83e5592edd4.png">

This will fix the _date/time field value out of range_ error ^

Reply:
haha yes I just fixed them ^^